pkgs/tools/networking/ntopng/default.nix

```
@@ -1,41 +1,44 @@
-{ stdenv, fetchurl, libpcap,/* gnutls, libgcrypt,*/ libxml2, glib
-, geoip, geolite-legacy, sqlite, which, autoreconfHook, git
-, pkgconfig, groff, curl, json_c, luajit, zeromq, rrdtool
+{ stdenv, fetchurl, fetchFromGitHub, libpcap,/* gnutls, libgcrypt,*/ libxml2, glib
+, geoip, geolite-legacy, sqlite, which, autoreconfHook, git, mariadb, readline80
+, pkgconfig, groff, curl, json_c, luajit, zeromq, rrdtool, libmaxminddb
 }:
 
 # ntopng includes LuaJIT, mongoose, rrdtool and zeromq in its third-party/
 # directory, but we use luajit, zeromq, and rrdtool from nixpkgs
 
-stdenv.mkDerivation rec {
-  name = "ntopng-2.0";
+let
+  ndpi = fetchurl {
+    url = "https://github.com/ntop/nDPI/archive/2.6.tar.gz";
+    sha256 = "07prvgdbs09kyq83s5n8j0mdqrc7jwl0acr0k43ihnrq824vdpzg";
+  };
+in stdenv.mkDerivation rec {
+  version = "3.8";
+  name = "ntopng-${version}";
```

```suggestion
  pname = "ntopng";
```

`name` will be defined automatically